sdk/keyvault/azure-security-keyvault-certificates/src/test/resources/session-records/cancelCertificateCreationOperation.json

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
I'd rather re-record these tests than just copy another similar test's recording. If you need help setting this up I can give you a hand. If you are unable to, I think eventually I should be able myself :)